src/veh_interact.cpp

```diff
@@ -558,6 +577,13 @@ bool veh_interact::can_install_part(int msg_width){
                             dif_eng);
     }
 
+    if (dif_steering > 0) {
+        engine_string = string_format(
```

Comment:
Right, yeah. Actually we could probably get away with one less var here if we never have "steerable engines"